### ISSUE TITLE
Specify network other than sentry-self-hosted_default

### ISSUE THREAD

**Issue by drmrbrewer** [getsentry/self-hosted#1302]
### Problem Statement

Is it possible to specify a network other than `sentry-self-hosted_default`?  I need to specify the name of an existing network instead.

Thanks.

**Comment by drmrbrewer** [comment]
Since the sentry install process just uses docker-compose, it looks like adding the following lines to the `docker-compose.yml` file does the trick (see e.g. [here](https://stackoverflow.com/a/42750687/4070848)):

    networks:
      default:
        external:
          name: my-overlay-network

This attaches all of the sentry containers to the existing `my-overlay-network` rather than creating a separate network and using that.

**Comment by aminvakil** [comment]
Beware that sentry creates a network in this line too:

https://github.com/getsentry/self-hosted/blob/79292a3e411b83df559367319934a678f9da7ddf/install/wrap-up.sh#L9-L10

**Comment by drmrbrewer** [comment]
@aminvakil thanks for the heads-up.  I had spotted that line before, when searching for where the network name was specified in the sentry installation config files, but forgot about it after I realised that the network name would mainly be specified via docker-compose (with a default name being based on `COMPOSE_PROJECT_NAME` as per [here](https://docs.docker.com/compose/networking/)).  In fact in my case I don't think I ever get to that part you refer to, because it is only executed when `MINIMIZE_DOWNTIME` is set, but maybe in future I will indeed need to do so.
